### ISSUE TITLE
Feature request: Would love to see a compact list of items

### ISSUE THREAD

**Issue by gabek** [simplerocket-llc/dear-cashew#59]
In order to fit more on the screen I'd love to see a compact list view.  I don't need to see the user avatar or the "Opened n minutes ago" item and would instead prefer an option to see more items at once.

**Comment by hishboy** [comment]
Thanks for the feedback. I’ll definitely look into it and get back to you as soon as I have something

**Comment by hishboy** [comment]
@gabek, what would you like to generally see in compact mode?

**Comment by gabek** [comment]
I'm coming from another tool that's more spreadsheety.
<img width="1404" alt="screen shot 2016-08-16 at 9 58 32 am" src="https://cloud.githubusercontent.com/assets/414923/17708058/10ef88c0-6398-11e6-916e-4ef0e0d53ef5.png">

I've found that the subject, assignee, and labels could be the most useful in this case.  Though I understand that this is a different design than the barebones tool I was using :)  Thanks for being open to the suggestion!

**Comment by hishboy** [comment]
Here’s what I got so far (see screenshot below). Still working out some bugs but you’ll be able to use a Preference setting to switch between `Standard` mode or `Classic` mode`. Classic mode is basically spreadsheet-style.

I ping you once Cashew `v1.9` is available in the app store.

![image](http://content.cashewapp.io/github/hishboy/20160817053430UTC/dbbbe381-7c28-46f5-9c89-c2eee5bae142.png)

**Comment by gabek** [comment]
That's so cool, thank you for knocking that out!  Looking forward to using this in 1.9, it really looks great.

**Comment by ricburton** [comment]
Love the mac style bar!

Tabs would be dope too: creating modern cocoa apps

Richard

**Comment by ricburton** [comment]
https://developer.apple.com/videos/play/wwdc2016/239/

Richard

**Comment by hishboy** [comment]
@ricburton Thanks. Tabs would be nice. Probably going to wait on that until Sierra is release. I created a separate issue for it here #60

**Comment by hishboy** [comment]
@gabek - `1.9` is now in the app store. Grab it and let me know if you encounter any issues

**Comment by gabek** [comment]
@hishboy This works fantastically, thank you for putting that together!  Cashew really is the best, and fastest way to navigate Github Issues.  You're awesome.
